Overview.src.html

```diff
@@ -2663,6 +2685,30 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   <p><span title="Read a request">Read <var>request</var></span>.
 
  <li>
+  <p>Let <var>strategy</var> be an object. The user agent may choose any object.
```

Comment:
Filed https://github.com/whatwg/streams/issues/427 to track this in the stream spec.